### ISSUE TITLE
RELEASE "TWISTED" 2.4.0!!!!

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1747]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1747|
|Type|enhancement|
|Created|2006-05-25 01:33:03Z|

YAY

<details><summary>Searchable metadata</summary>

```
trac-id__1747 1747
type__enhancement enhancement
reporter__radix radix
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1148520783000000 1148520783000000
changetime__1148763361000000 1148763361000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

There were minor changes to setup.py in [branches/releases/twisted-2.4.x-1747](branches/releases/twisted-2.4.x-1747)

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

Looks good to me.  As foom suggested on IRC, a comment about "cd Project; setup.py install" replacing "setup.py <project> install" would be good, if that's not already documented elsewhere.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @radix|
|-|-|

Looks good.  Merge away.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16950]) merge the shrapnel from the Twisted 2.4.0 release.

Author: radix
Reviewers: spiv, exarkun
fixes [#1747](https://github.com/twisted/twisted/issues/1747)

This fixes setup.py when used in the context of a sumo release, 
and changes the README a bit.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
